### PR TITLE
Add CoNLL 2000

### PR DESCRIPTION
**Pull request by Ayushk4**
As of now, I can't use the same struct and functions as `CoNLL 2003`, since the two datasets have different MultiResolutionIteratos `levelname_map`. Is there any suggestion that you may have regarding this?

- [X] Support for CoNLL 2000
- [X] Tests
- [X] Docs

### CODE REVIEW

**Comment by Ayushk4** [comment]
@oxinabox Please review when you have time. This PR is done from my side.

File: src/CoNLL2000.jl [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    end
+    isempty(sents[end]) && deleteat!(sents, lastindex(sents))
+
+    return context
+end
+
+function load(corpus::CoNLL2000, file="train")
+    file == "train" && return parse_conll2000file(corpus.trainpath)
+    file == "test" && return parse_conll2000file(corpus.testpath)
+    file == "dev" && throw("Dev set not Available in $year.")
```
Comment (oxinabox) [inline src/CoNLL2000.jl]:
```suggestion
    file == "dev" && throw("Dev set not available in $year.")
```

**Comment by Ayushk4** [comment]
Once #29 is merged, I will also address the exporting of `part_of_speech` API in the docs.

**Comment by Ayushk4** [comment]
Merging this.

**Pull request merged**